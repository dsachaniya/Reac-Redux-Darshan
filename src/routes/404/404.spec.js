import React from 'react';
import { shallow } from 'enzyme';
import RouteNotFound from './404';

describe('<RouteNotFound />', () => {
  it('Rendering', () => {
    const wrapper = shallow(<RouteNotFound />);
    expect(wrapper).toMatchSnapshot();
  });
});
