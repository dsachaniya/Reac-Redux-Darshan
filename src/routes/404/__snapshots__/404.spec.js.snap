// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RouteNotFound /> Rendering 1`] = `
<section>
  <h1>
    Oops...! 404 Route Not Found
  </h1>
</section>
`;
